Library/Formula/cccc.rb

```diff
@@ -0,0 +1,47 @@
+class Cccc < Formula
+  desc "C and C++ Code Counter"
+  homepage "http://sourceforge.net/projects/cccc/"
+  url "https://downloads.sourceforge.net/project/cccc/cccc/3.1.4/cccc-3.1.4.tar.gz"
+  sha256 "27b3eca9a83a66799332363a80cc4bcd6db9869eddbda1a59a25cdace3ff4dbf"
+  patch :DATA
+
+  def install
+    # "make cccc" will fail if parallelized.
+    ENV.deparallelize
```

Comment:
Can you please submit this issue to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!